tests/jepsen/run.sh

```diff
@@ -58,7 +58,19 @@ fi
 
 if [ ! -d "$script_dir/jepsen" ]; then
     # TODO(deda): install apt get docker-compose-plugin on all build machines.
-    git clone https://github.com/jepsen-io/jepsen.git -b "$JEPSEN_VERSION" "$script_dir/jepsen"
+    git clone http://mgdeps-cache:8000/git/jepsen.git -b "$JEPSEN_VERSION" "$script_dir/jepsen"
+    if [ "$?" -ne 0 ]; then
+        git clone https://github.com/jepsen-io/jepsen.git -b "$JEPSEN_VERSION" "$script_dir/jepsen"
+        if [ "$?" -ne 0 ]; then
+            ERROR "Failed to clone Jepsen from the public repo."
+            exit 1
+        else
+            echo "Successfully cloned Jepsen from the public repo."
+        fi
+    else
+      echo "Successfully cloned Jepsen from the mgdeps-cache."
+    fi
+
```

Comment:
This has some problems, if mgdeps clone fails the script will exit and the same goes for github repo link, use this code block instead. Might seem a bit ugly but it will do the job
```
echo "Cloning Jepsen ..."
git clone http://mgdeps-cache:8000/git/jepsen.git -b "$JEPSEN_VERSION" "$script_dir/jepsen" &> /dev/null \
|| git clone https://github.com/jepsen-io/jepsen.git -b "$JEPSEN_VERSION" "$script_dir/jepsen" &> /dev/null
```